pkg/steps/release/create_release.go

```diff
@@ -193,24 +193,48 @@ func (s *assembleReleaseStep) Run(ctx context.Context, dry bool) error {
 		return s.importFromReleaseImage(ctx, dry, providedImage)
 	}
 
-	stable, err := s.imageClient.ImageStreams(s.jobSpec.Namespace).Get(streamName, meta.GetOptions{})
-	if err != nil {
-		if errors.IsNotFound(err) {
+	var stable *imageapi.ImageStream
+	var cvo string
+	cvoResolved := false
+	cliResolved := false
+	// waiting for importing the images
+	// 2~3 mins: build01 on aws imports images from api.ci on gcp
+	log.Printf("waiting for importing cluster-version-operator and cli ...")
+	if err := wait.Poll(10*time.Second, 5*time.Minute, func() (bool, error) {
+		stable, err = s.imageClient.ImageStreams(s.jobSpec.Namespace).Get(streamName, meta.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		if !cvoResolved {
```

Comment:
nit: the only thing `resolvePullSpec` does is pull data from the `stable` stream, so we don't really get anything from doing this conditionally -- it's not costly. This could be simpler:

```
cvoPullSpec, cvoExists := resolvePullSpec(stable, "cluster-version-operator", true)
_, cliExists := resolvePullSpec(stable, "cluster-version-operator", true)
return cvoExists && cliExists, nil
```

Reply:
Updated